doc/md/_a6_Getting-Started-External-SDRAM.md

```diff
@@ -0,0 +1,74 @@
+# Getting Started - External SDRAM
+
+## Usage
+
+The Daisy comes with 64MB of external memory in addition to it's 1MB of internal memory
+
+This is extremely valuable for DSP effects like delays, sample-based synthesis, and more.
+
+The external memory, however, does require a little bit of special handling to use.
+
+Let's first look at a normal array of floats. Something we might use for a buffer or delay:
+
+```cpp
+float my_buffer[1024];
+```
+
+So that's 1024 floats in an array within our normal memory. Totalling 4kB of memory. This will easily fit in any of the memory regions available on the Daisy.
+
+You can declare that anywhere and do anything with it.
+
+Well, the SDRAM is a little bit different.
+
+Due to the way the SDRAM is handled, an object located within it has to be created globally (at least the easy way), and it can't have a constructor of any importance (a C++ construct that happens before the SDRAM is fuly initialized).
+
+Now, for most things (like buffers, arrays, etc.) this doesn't matter much.
+
+So, let's make the same array again, but in the SDRAM:
+
+```cpp
+float __attribute__(section((".sdram_bss"))) my_buffer[1024];
+```
+
+Now, that is a lot of stuff to type, and pretty hard to remember. So we made a shorthand macro to make it a bit easier:
+
+```cpp
+float DSY_SDRAM_BSS my_buffer[1024];
+```
+
+The first, longform example is the spelled out gcc attribute that says what memory section to put the array in. This can be useful on big projects when using one of the many other available memory sections within the STM32H7 processor.
```

Comment:
```suggestion
The first, longform example is the spelled out compiler attribute that says what memory section to put the array in. This can be useful on big projects when using one of the many other available memory sections within the STM32H7 processor.
```